### ISSUE TITLE
install by composer minimum-stability exception

### ISSUE THREAD

**Issue by dennisbaum** [getkirby/staticache#5]
i guess while the plugin is in dev-mode installation by `composer require getkirby/staticache`results in the following error:

```
  [InvalidArgumentException]                                                                                                                             
  Could not find a version of package getkirby/staticache matching your minimum-stability (stable). Require it with an explicit version constraint allo  
  wing its desired stability. 
```

`composer require getkirby/staticache 'dev-main'`
or set  
`minimum-stability`in your composer.json to `dev`  
is working
```
{
  "minimum-stability": "dev",
  "require": {
```

maybe you could mention that in the readme?

**Comment by bnomei** [comment]
please just make it v1.0.0. the plugin is imho stable enough

**Comment by bnomei** [comment]
nevermind its not stable

**Comment by lukasbestle** [comment]
✅ Fixed by the upcoming version 1.0.0
